<?xml version="1.0"?>
<launch>
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>
  <node pkg="nodelet" type="nodelet" name="NodeletRosbag" args="load nodelet_rosbag/NodeletRosbag standalone_nodelet" output="screen">
    <rosparam command="load" file="$(find nodelet_rosbag)/config/rosbag.yaml" />
  </node>
</launch>
